recipe/meta.yaml

```diff
@@ -103,8 +103,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - khallock
-    - marylhaley
-    - david-ian-brown
-    - ocefpaf
-    - xylar
+    - erogluorhan
```

Comment:
Please do not remove folks without their consent. I certainly do not want to be removed as the original author of this recipe.

Reply:
Restored the maintainers.